LICENSE

```diff
@@ -198,4 +198,4 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
```

Comment:
I can't tell why this shows up in the diff. Could you check there's no weird non-printable characters here or sth?

Reply:
I think I just added a new line